setup.py

```diff
@@ -37,7 +44,7 @@
             'codecov',
             'flake8',
             'isort',
-            'pytest>=3.2',
+            'pytest>=4.6',
```

Comment:
What has changed that this is needed?

Reply:
as you know the pinned tests fail if we don't do this, same as with codegen, it's due to the latest pytest-cov (which is the line below it) now requiring a higher pytest version.